README.md

```diff
@@ -56,346 +15,25 @@ Currently, the project provides several single-purpose tools, all exposed as she
 - The [Compare Model Tool](site/compare.md) (`compareModel`) compares two model files.
 - The [Prepare Model Tool](site/prepare.md) (`prepareModel`) prepare model files for deploying to WebLogic Kubernetes Operator environment.
 - The [Extract Domain Resource Tool](site/kubernetes.md) (`extractDomainResource`) generates a domain resource YAML for use with the Oracle WebLogic Server Kubernetes Operator.
+- The [Model Help Tool](site/model_help.md) (`modelHelp.sh`) provides information about the folders and attributes that are valid for sections and folders of a domain model.
 
 As new use cases are discovered, new tools will likely be added to cover those operations but all will use the metadata model to describe what needs to be done.
```

Comment:
the variable injector tool also a "tool". should it be under this section?

Reply:
If so, then yes, it should be listed here.

Reply:
There's some confusion here (not new) as the page for "Variable Injector Tool" (site/variable_injection.md) is also the page for "Variable Injection" under "Tool Configuration". We could link that page from both places for now, but long-term, I think it should be split into two topics, the stand-alone tool and the configuration. The configuration is relevant for Discover, Variable Injector Tool, and Prepare Model Tool.

Reply:
If we link here, we could just use the intro sentence from that page: 
"The Variable Injector Tool is used to tokenize a model with variables."